src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -296,12 +299,12 @@ public abstract class Controller
     protected String keyspaceName;
     protected String tableName;
 
-    protected final int baseShardCount;
+    protected int baseShardCount;
 
     protected final long targetSSTableSize;
-    protected final double sstableGrowthModifier;
+    protected double sstableGrowthModifier;
```

Comment:
similar to `scalingParameters`, we can have thread-safe issue with these non-final params